src/pages/Admin/AdminCohortsPage/AddNewCohortForm.tsx

```diff
@@ -18,7 +18,7 @@ export const AddNewCohortForm = () => {
       </InputField>
       <InputField htmlFor="name">
         <span>Start Date</span>
-        <Input type="text" id="start-date" placeholder="DD-MM-YYYY" />
+        <Input type="date" id="start-date" placeholder="DD-MM-YYYY" />
```

Comment:
this should be left as `text` to avoid conflict, cause I'm changing it to `date` in my PR 😄

Reply:
Okay, will do!